app/assets/stylesheets/general/_form.scss

```diff
@@ -312,3 +316,75 @@ input[type="file"] {
   font-size: $font-size-small;
   color: $text-color-dark;
 }
+
+.form-tags-wrapper {
```

Comment:
Would it make more sense to move all `form-tags` classes to their own scss file?

Reply:
It probably makes sense at this point to be fair.